### ISSUE TITLE
gptfdisk: Unable to generate a proper UUID

### ISSUE THREAD

**Issue by fanck0605** [openwrt/packages#24075]
Maintainer: unknown
Environment: (put here arch, model, OpenWrt version)

arch: x86_64
version: v23.05.3

src-git packages https://git.openwrt.org/feed/packages.git^063b2393cbc3e5aab9d2b40b2911cab1c3967c59
src-git luci https://git.openwrt.org/project/luci.git^b07cf9dcfc37e021e5619a41c847e63afbd5d34a
src-git routing https://git.openwrt.org/feed/routing.git^648753932d5a7deff7f2bdb33c000018a709ad84
src-git telephony https://git.openwrt.org/feed/telephony.git^86af194d03592121f5321474ec9918dd109d3057

Description:

I want to recreate the partition by **`sgdisk`** command, but it print the warning log.

```
# sgdisk -og -N1 -p /dev/sdb
Warning! Unable to generate a proper UUID! Creating an improper one as a last
resort! Windows 7 may crash if you save this partition table!
```

It seems that the uuid module does not works well. Although the partitions can be formatted and mounted normally on OpenWrt, i'm not sure if there are any potential issues in the partition I created. Is there anyone can fix it. Thanks!

@krant @httpstorm @1715173329 @neheb

**Comment by krant** [comment]
@fanck0605 gptfdisk 1.0.10 has it fixed. If you want it to become available on OpenWrt 23.05 ASAP - I'd suggest you to create PR with https://github.com/openwrt/packages/commit/ad6344d0455038e374f57a7fb15d3d1ace8d889b cherry-picked to openwrt-23.05 branch.

**Comment by neheb** [comment]
backported.
